jsonschema-validation.xml

```diff
@@ -799,14 +799,48 @@
 
             <section title="Defined formats">
 
-                <section title="date-time">
-                    <t>
-                        This attribute applies to string instances.
+                <section title="Dates and times">
+                    <t>
+                        These attributes apply to string instances.
+                    </t>
+                    <t>
+                        Date and time format names are derived from
+                        <xref target="RFC3339">RFC 3339, section 5.6</xref>.
+                    </t>
+                    <t>
+                        Implementations supporting formats SHOULD implement support for
+                        the following attributes:
+                        <list style="hanging">
+                            <t hangText="date-time">
+                                A string instance is valid against this attribute if it is
+                                a valid representation according to the "date-time" production.
+                            </t>
+                            <t hangText="date">
+                                A string instance is valid against this attribute if it is
+                                a valid representation according to the "full-date" production.
+                            </t>
+                            <t hangText="time">
+                                A string instance is valid against this attribute if it is
+                                a valid representation according to the "full-time" production.
+                            </t>
+                        </list>
                     </t>
                     <t>
-                        A string instance is valid against this attribute if it is a valid date
-                        representation as defined by <xref target="RFC3339">RFC 3339, section
-                        5.6</xref>.
+                        Implementations MAY support additional attributes using the other
+                        production names defined in that section.  If implementing
+                        "full-date" or "full-time", the corresponding short form ("date"
```

Comment:
The beginning of "If implementing..." is not clear to me. Did you mean something like `If "full-date" or "full-time" is implemented, the corresponding...`?